astropy/stats/funcs.py

```diff
@@ -718,7 +724,8 @@ def poisson_conf_interval(n, interval='root-n', sigma=1, background=0,
     return conf_interval
 
 
-def median_absolute_deviation(a, axis=None):
+@deprecated_renamed_argument('a', 'data', '1.3')
+def median_absolute_deviation(data, axis=None, ignore_nan=False):
```

Comment:
I'll have to check how deprecated_renamed_argument works, but I have a feeling that 'a' should be changed in the doc_string as well to 'data'